content/04-guides/02-deployment/00-deploying-to-heroku.md

```diff
@@ -0,0 +1,283 @@
+npm install
+```
+
+> The `Procfile` tells Heroku the command needed to start the application, i.e. `npm start`
+
+## 2. Create a Git repository for the application
+
+In the previous step, you downloaded the code. In this step, you will create a repository from the code so that you can push it to Heroku for deployment.
+
+To do so, run `git init` from the source code folder:
+
+```no-lines
+git init
+> Initialized empty Git repository in /Users/alice/prisma-heroku/.git/
+```
+
+With the repository initialized, add and commit the files:
+```no-lines
+git add .
+git commit -m 'Initial commit'
+```
+
+**Checkpoint:** `git log -1` should show the commit:
+```no-lines
+git log -1
+commit 895534590fdd260acee6396e2e1c0438d1be7fed (HEAD -> master)
+```
+
+
+## 3. Heroku CLI login
+
+Make sure you're logged in to Heroku with the CLI:
+
+```no-lines
+heroku login
+```
+
+This will allow you to deploy to Heroku from the terminal.
+
+**Checkpoint:** `heroku auth:whoami` should show your username:
+
+```no-lines
+heroku auth:whoami
+> your-email
+```
+
+## 4. Create a Heroku app
+
+To deploy an application to Heroku, you need to create an app. You can do so with the following command:
+
+```no-lines
+heroku apps:create your-app-name
+```
+
+> Use a unique name of your choice instead of `your-app-name`.
+
+**Checkpoint:** You should see the URL and the repository for your Heroku app:
+
+```no-lines
+heroku apps:create your-app-name
+> Creating ⬢ your-app-name... done
+> https://your-app-name.herokuapp.com/ | https://git.heroku.com/your-app-name.git
+```
+
+## 5. Connect your local code repository to Heroku for deployment
+
+To deploy your application, you need to add the git remote that Heroku created to your local repository.
+
+You can do so with the following Heroku CLI command:
+
+```no-lines
+heroku git:remote --app your-app-name
+```
+
+**Checkpoint:** `git remote -v` should show the Heroku git remote for your application:
+
+```no-lines
+heroku	https://git.heroku.com/your-app-name.git (fetch)
+heroku	https://git.heroku.com/your-app-name.git (push)
+```
+
+## 6. Add a PostgreSQL database to your application
+
+Heroku allows your to provision a PostgreSQL database as part of an application.
+
+Create the database with the following command:
+
+```no-lines
+heroku addons:create heroku-postgresql:hobby-dev
+```
+
+**Checkpoint:** To verify the database was created you should see the following:
+
+```no-lines
+Creating heroku-postgresql:hobby-dev on ⬢ your-app-name... free
+Database has been created and is available
+ ! This database is empty. If upgrading, you can transfer
+ ! data from another database with pg:copy
+Created postgresql-parallel-73780 as DATABASE_URL
+```
+
+> Note: Heroku will automatically set the `DATABASE_URL` environment variable when the app is running on Heroku. Prisma will use this environment variable because it's declared in the _datasource_ block of the Prisma schema (`prisma/schema.prisma`) with `env("DATABASE_URL")`.
```

Comment:
For notes like this, we've typically put **Note** in bold:

> Note**:** Heroku will automatically set the `DATABASE_URL` environment variable when the app is running on Heroku. Prisma will use this environment variable because it's declared in the _datasource_ block of the Prisma schema (`prisma/schema.prisma`) with `env("DATABASE_URL")`.

(you might also adjust the other "notes" written in markdown quotes to follow the same "pattern")